dask_sql/physical/rex/core/call.py

```diff
@@ -613,6 +614,41 @@ def extract(self, what, df: SeriesOrScalar):
             raise NotImplementedError(f"Extraction of {what} is not (yet) implemented.")
 
 
+class ToTimestampOperation(Operation):
+    def __init__(self):
+        super().__init__(self.to_timestamp)
+
+    def to_timestamp(self, df, format):
+        default_format = "%Y-%m-%d %H:%M:%S"
+        # Remove double and single quotes from string
+        format = format.replace('"', "")
+        format = format.replace("'", "")
+
+        # TODO: format timestamps for GPU tests
+        if "cudf" in str(type(df)):
+            if format != default_format:
+                raise RuntimeError("Non-default timestamp formats not supported on GPU")
+            if df.dtype == "object":
+                return df
+            else:
+                nanoseconds_to_seconds = 10**9
+                return df * nanoseconds_to_seconds
+        # String cases
+        elif type(df) == str:
+            return datetime.strptime(df, format)
```

Comment:
Is it possible to return a NumPy datetime64 here instead.

Reply:
From what I tried, `np.datetime64` can't use the `format` param the same way `datetime.strptime` can. I could wrap it in a NumPy datetime64, like `np.datetime64(datetime.strptime(df, format))`, if desired, though.

Reply:
Yeah I think that works. Just suggested this change to standardize on all python return types for user requesting sql timestamp to be inline with datetime64 as specified here: https://github.com/dask-contrib/dask-sql/blob/5440effe0d26c746650c4e801c7f846efd17f5c9/dask_sql/mappings.py#L79-L83

Reply:
Ok, updated. Thanks!